### PR TITLE
rewrite vertex cache UT

### PR DESCRIPTION
**Pull request by panda-sheep**
As title.

### CODE REVIEW

**Review by monadbobo — APPROVED** [review]
LGTM.

**Review by critical27 — APPROVED** [review]
Good job

File: src/storage/test/CMakeLists.txt [hunk 1]
```diff
@@ -442,6 +442,22 @@ nebula_add_executable(
...
+    LIBRARIES
+        ${ROCKSDB_LIBRARIES}
+        ${THRIFT_LIBRARIES}
+        proxygenlib
+        wangle
+        gtest
+)
+
+
 #nebula_add_test(
```
[the remark this answers is not in the document]
Comment (panda-sheep) [reply src/storage/test/CMakeLists.txt]: Fixed.  all comment UTs have handled.

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Pull request merged**